ext/web/02_timers.js

```diff
@@ -25,8 +25,13 @@
   const { reportException } = window.__bootstrap.event;
   const { assert } = window.__bootstrap.infra;
 
+  const buf = new ArrayBuffer(8)
+  const u32 = new Uint32Array(buf)
+
   function opNow() {
-    return ops.op_now();
+    ops.op_now_new(buf);
+    console.log("buf1", buf, u32, (u32[1] & 0xff));
+    return u32[0] + ((u32[1] & 0xff) * 0x100000000)
```

Comment:
i don't think we should include this logic without some further discussion - this was an experimental micro-optimization i made to see how fast things could go if we only returned an SMI from the function. if we did this, then the number representing seconds would reset to zero every 256 seconds and would require logic in the consumer to account for this.

Reply:
This was just an experiment, @littledivy came up with a better solution. I'm gonna close this PR